src/content/projects/an-archival-experience-celebrating-c-p-companys-50th-anniversary-–-milan-edition.md

```diff
@@ -0,0 +1,131 @@
+---
+templateKey: Project
+title: An archival experience celebrating C.P. Company’s anniversary – Milan edition
+intro: To round off C.P. Company’s 50th anniversary Cinquanta celebrations, we brought the interactive spatial archive we made for the brand last year to its home turf of Milan for the finnisage of their anniversary celebrations. Designed and built as a modular hybrid of physical and digital space, the exhibition premiered in Florence at Pitti Uomo 2021, then was installed in Darwen, UK, as part of the British Textile Biennale before travelling to BASE, a cultural center in Milan
+content:
+  - video:
```

Comment:
Try removing these empty `video` blocks. It should no longer be required to include an unused video element.